CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 This format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.16.3
+### Changed
+- Replace enum messages, don't concut them.
```

Comment:
nit (spelling): concat and "instead of appending more values to the repeated field"?